client/wfnews-war/src/main/angular/src/app/utils/index.ts

```diff
@@ -2,6 +2,7 @@ import { APP_BOOTSTRAP_LISTENER, Inject, InjectionToken, Type } from '@angular/c
 import { EffectSources } from '@ngrx/effects';
 import { SortDirection } from '@wf1/core-ui';
 import * as moment from 'moment';
+import { Moment } from 'moment';
```

Comment:
You have Moment imported directly above this. No need to load it twice.

Side note, preferably we don't use the moment library at all. It really isn't needed